src/ui/Facet/FacetSearchElement.ts

```diff
@@ -144,28 +153,19 @@ export class FacetSearchElement {
   }
 
   public setAsCurrentResult(toSet: Dom) {
-    this.currentResult && this.currentResult.removeClass('coveo-facet-search-current-result');
-    this.currentResult = toSet;
-    toSet.addClass('coveo-facet-search-current-result');
-    this.updateSelectedOption(toSet);
+    this.searchDropdownNavigator.setAsCurrentResult(toSet);
   }
 
-  public moveCurrentResultDown() {
-    let nextResult = this.currentResult.el.nextElementSibling;
-    if (!nextResult) {
-      nextResult = first(this.searchResults.children);
-    }
-    this.setAsCurrentResult($$(<HTMLElement>nextResult));
-    this.highlightAndShowCurrentResultWithKeyboard();
+  public get currentResult() {
+    return this.searchDropdownNavigator.currentResult;
   }
 
-  public moveCurrentResultUp() {
-    let previousResult = this.currentResult.el.previousElementSibling;
-    if (!previousResult) {
-      previousResult = last(this.searchResults.children);
-    }
-    this.setAsCurrentResult($$(<HTMLElement>previousResult));
-    this.highlightAndShowCurrentResultWithKeyboard();
+  public nextFocusableElement() {
```

Comment:
Question here: the old `public` methods were `moveCurrentResultDown`, and `moveCurrentResultUp`. I changed the names to ones that are more generic. That being said, this could be considered a breaking change since the old methods were `public` and are now no longer available.

On the other hand, I don't think anyone is using the FacetSearchElement class? so what do you think?

Atm, after writing this, I am leaning towards reverting the names to the old ones 🦆

Reply:
Yes if the name was not clear just document them, I would lean towards not touching the public methods name